addons/sys_intercom/vic3/vic3ffcs_dialogClasses.hpp

```diff
@@ -0,0 +1,62 @@
+class VIC3FFCS_RscBackground {
```

Comment:
```suggestion
class ACRE_VIC3FFCS_RscBackground {
```

Reply:
None of the radio dialogs have what you suggested. For coherence I would keep it like it is now.

Reply:
Let's hope no one else decides to ever use the same name? :P